src/synapse/azext_synapse/custom.py

```diff
@@ -106,9 +107,11 @@ def list_workspaces(cmd, client, resource_group_name=None):  # pylint: disable=u
         resource_group_name=resource_group_name) if resource_group_name else client.list()
 
 
-def create_workspace(cmd, client, resource_group_name, workspace_name, account_url, file_system, sql_admin_login_user,
-                     sql_admin_login_password, location, tags=None, identity_type="SystemAssigned", no_wait=False):
+def create_workspace(cmd, client, resource_group_name, workspace_name, storage_account, file_system,
+                     sql_admin_login_user, sql_admin_login_password, location, tags=None,
+                     identity_type="SystemAssigned", no_wait=False):

```

Comment:
I would suggest to use "--assign-identity" if this parameter is exposed. Reference `az sql server create`

Reply:
> I would suggest to use "--assign-identity" if this parameter is exposed. Reference `az sql server create`

removed this parameter.  Because now we only support this one value. It isn't a breaking change if we need to add the parameter back.